packages/formation-react/stories/form-controls.stories.mdx

```diff
@@ -0,0 +1,200 @@
+- When using another type of input will make answering more difficult. For example, birthdays and other known dates are easier to type in than they are to select from a calendar picker.
+- When users want to be able to paste in a response.
+
+#### When to consider something else
+
+- When users are choosing from a specific set of options.
+
+#### Guidance
+
+- The length of the text input provides a hint to users as to how much text to write. Do not require users to write paragraphs of text into a single-line input box; use a text area instead.
+- Text inputs are among the easiest type of input for desktop users but are more difficult for mobile users. Consider using specific `pattern` attributes or `type="tel"` or `type="number"` to trigger specific mobile keyboards.
+- Only show error validation messages or stylings after a user has interacted with a particular field.
+- Avoid using placeholder text that appears within a text field before a user starts typing. If placeholder text is no longer visible after a user clicks into the field, users will no longer have that text available when they need to review their entries. (People who have cognitive or visual disabilities have additional problems with placeholder text.)
+
+## Select box
+
+A select box allows users to select one option from a list.
+
+
+<Canvas>
+  <form class="usa-form">
+    <label for="options">Dropdown label</label>
+    <select class="usa-select" name="options" id="options">
+      <option value>- Select -</option>
+      <option value="value1">Option A</option>
+      <option value="value2">Option B</option>
+      <option value="value3">Option C</option>
+    </select>
+  </form>
+</Canvas>
+
+### Accessibility
+
+- Make sure your select box has a label. Don’t replace it with the default menu option (for example, removing the “State” label and just having the dropdown read “Select a state” by default).
+- Don’t use JavaScript to automatically submit the form (or do anything else) when an option is selected. Auto-submission disrupts screen readers because they select each option as they read them.
+
+### Usability
+
+#### When to use
+
+- Use sparingly — only when a user needs to choose from about seven to 15 possible options and you have limited space to display the options.
+
+#### When to consider something else
+
+- If the list of options is very short. Use radio buttons instead.
+- If the list of options is very long. Let users type the same information into a text input that suggests possible options instead.
+- If you need to allow users to select more than one option at once. Users often don’t understand how to select multiple items from select boxes. Use checkboxes instead.
+- If you are intending to provide navigation. To help users navigate to a section on the same page or another page in the site, use a [text link](/design/typography#links) or the [Sidenav component](/components/sidenav).
```

Comment:
These links don't work :upside_down_face:

Reply:
They should once this lives on design.va.gov.

Reply:
![image](https://user-images.githubusercontent.com/2008881/100787774-fdd4f580-33c8-11eb-85d8-9235535ba800.png)

Reply:
Putting a `..` in front of the leading `/` seems to get it out of storybook mode:

![image](https://user-images.githubusercontent.com/2008881/100788484-fc57fd00-33c9-11eb-8958-db6862bbe3ee.png)